app/_hub/moesif/kong-plugin-moesif/index.md

```diff
@@ -40,6 +52,10 @@ kong_version_compatibility: # required
     #incompatible:
   enterprise_edition: # optional
     compatible:
+      - 2.1.x-x
+      - 1.5.x-x
+      - 1.3.x-x
```

Comment:
```suggestion
      - 1.3-x-x
```

@lena-larionova is the above extra x how we should be formatting versions for KE and CE? I have my plugin version compatibility PR in progress.

Reply:
I am not seeing these show up in the side panel 

![image](https://user-images.githubusercontent.com/3756245/90649615-8b1c8180-e200-11ea-902b-d9c81efc127a.png)

Reply:
> @lena-larionova is the above extra x how we should be formatting versions for KE and CE? I have my plugin version compatibility PR in progress.

No - the formatting of these versions should match what's used in the site URLs, or the doc site version dropdown. In this case, the versions should be 1.3-x, 1.5.x, and 2.1.x